### PR TITLE
Clarify policy support for attendance report

### PR DESCRIPTION
**Pull request by bingqianshen**
Attendance report policy only cover the post-meeting reports, for the in-meeting reports, they are not covered by these settings, update the official docs to clarify it.

### CODE REVIEW

File: Teams/settings-policies-reference.md [hunk 1]
```diff
@@ -315,8 +315,8 @@ Meeting policies are used to control what features are available in meetings org
...
 |Attendance report|Everyone, unless organizers opt-out|This setting allows meeting organizers the ability to see the toggle that turns on or off Attendance Reports within Meeting options.|
-|Who is in the report|Everyone, but participants can opt out|This setting controls whether participants in the meeting can opt in or out of offering their attendance information in the Attendance Report.|
-|Attendance summary|Show everything|This setting controls whether to show attendance time information - such as join times, leave times, and in-meeting duration - for each meeting participant.|
+|Who is in the report|Everyone, but participants can opt out|This setting controls whether participants in the meeting can opt in or out of offering their attendance information in the Attendance Report. Only post-meeting report supported|
+|Attendance summary|Show everything|This setting controls whether to show attendance time information - such as join times, leave times, and in-meeting duration - for each meeting participant. Only post-meeting report supported|
```
Comment (JohanFreelancer9) [inline Teams/settings-policies-reference.md:319]:
```suggestion
|Who is in the report|Everyone, but participants can opt out|This setting controls whether participants in the meeting can opt in or out of offering their attendance information in the Attendance Report. Only the post-meeting report is supported.|
|Attendance summary|Show everything|This setting controls whether to show attendance time information - such as join times, leave times, and in-meeting duration - for each meeting participant. Only the post-meeting report is supported.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bingqianshen, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by bingqianshen** [comment]
@JohanFreelancer9 , could you please help to merge this PR or share how I could apply permissions to contribute to the doc? Thanks!

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 , could you please help to merge this PR or share how I could apply permissions to contribute to the doc? Thanks!

@bingqianshen, I only did a copy pass. Your changes will now be reviewed by the document owner or manager, and if approved, they will merge your changes into the live document. So at this time, nothing more is required from you. Many thanks!

**Comment by scanum** [comment]
@mkbond007 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**